src/nccl_ofi_rdma.cpp

```diff
@@ -4064,7 +4064,14 @@ static inline void free_rdma_send_comm(nccl_net_ofi_rdma_send_comm_t *s_comm) {
     }
 }
 
-static int send_comm_destroy(nccl_net_ofi_rdma_send_comm_t *s_comm)
+
+/**
+ * Destroy a send communicator
+ *
+ * @param release_ep: whether to also release the endpoint associated with the
+ * 		      communicator
+ */
+static int send_comm_destroy(nccl_net_ofi_rdma_send_comm_t *s_comm, bool release_ep)
```

Comment:
I don't like this approach.  I'd rather we manage the reference counting properly here.